images/csp_violation_report_service/app/requirements.txt

```diff
@@ -0,0 +1,7 @@
+aws-lambda-powertools==1.26.0
+masonite>=4.0,<5.0
+masonite-orm>=2.0,<3.0
```

Comment:
Could we pin both of these to a specific version to cut down on any weirdness during version bumps and the build process?